src/EnergyPlus/HVACFourPipeBeam.cc

```diff
@@ -1485,8 +1494,21 @@ namespace FourPipeBeam {
         this->supAirHeatingEnergy = this->supAirHeatingRate * ReportingConstant;
 
         this->primAirFlow = this->mDotSystemAir / DataEnvironment::StdRhoAir;
+        
+        this->CalcOutdoorAirVolumeFlowRate();
     }
 
+    void HVACFourPipeBeam::CalcOutdoorAirVolumeFlowRate()
+    {
+        // calculates zone outdoor air volume flow rate using the supply air flow rate and OA fraction
+        if (this->airLoopNum > 0) {
+            this->OutdoorAirFlowRate = (DataLoopNode::Node(this->airOutNodeNum).MassFlowRate / DataEnvironment::StdRhoAir) * DataAirLoop::AirLoopFlow(this->airLoopNum).OAFrac;
+        } else {
+            this->OutdoorAirFlowRate = 0.0;
+        }
+    }
+
+
```

Comment:
The new member function for calculating and setting value to *OutdoorAirFlowRate* report variable for FourPipeBeam air terminal.